### DISCUSSION TITLE
wkhtmltopdf issue

### DISCUSSION THREAD

**Discussion by jaykishanvmer** [serenity-is/Serenity#6083] [Help] [answered]
I am getting this empty pdf file with only headers are displayed on my project.

So I created a fresh new project and latest .net core 5.0 last known version.
I clicked: Basic Samples -> Grids -> Initial Value for Quick Flt. -> click grid pdf
it still resulted as show in screenshot below, please help!

Using  wkhtmltopdf 0.12.6 version, issue may not be related to wkhtmltopdf. I am just stuck.

![image](https://user-images.githubusercontent.com/61444129/141261044-b4347b41-589a-41d7-ab28-0bea0cdc293b.png)

**Comment by VictorTomaili** [comment, marked as answer]
[In this sample](https://serenity.is/demo/BasicSamples/InitialValuesForQuickFilters) when you change the filter values what has some data, and when you click the pdf icon near the order id, you can see the pdf generation is working as expected. 

Something is wrong with your environment. Please check and debug your code, you can clone the [Serene](https://github.com/serenity-is/Serene) source and debug it step by step to understand what's wrong in your environment.

**Comment by jaykishanvmer** [comment]
@VictorTomaili  First statement, I understand completely.

Second statement, I did clone serene and startsharp source code both as well, both results same empty pdf. I am unable to fix this problem. If you find solution, let me know, thanks in advance. I understand you know correctly, I will check my environment.

I will update if I find solution, so that some one phase similar issue it helps.

**Reply by jaykishanvmer** [reply]
@VictorTomaili  
I tried on other machine with serene normal version, it worked like charm. I am trying to figure it out on startsharp machine.

**Reply by jaykishanvmer** [reply]
@VictorTomaili 

I did uninstalled all the softwares, reinstalled vs2022 after completely all libraries with unrelated programs and features were uninstalled, all .net core libraries in it's path, all unused environment variables, npm, nodejs, typescript cache and reinstalled but still it didn't help. Only solution I got was to format my machine, it worked like charm.

**Reply by VictorTomaili** [reply]
I'm sorry about that. But anyway, currently you have a clean new environment.
